SharpBucketTests/V2/EndPoints/IssuesResourceTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Linq;
+using System.Net;
+using NUnit.Framework;
+using SharpBucket.V2;
+using SharpBucket.V2.EndPoints;
+using SharpBucket.V2.Pocos;
+using Shouldly;
+
+namespace SharpBucketTests.V2.EndPoints
+{
+    [TestFixture]
+    internal class IssuesResourceTests
+    {
+        private IssuesResource ExistingRepository { get; set; }
+
+        private IssuesResource NotExistingRepository { get; set; }
+
+        [OneTimeSetUp]
+        protected void Init()
+        {
+            ExistingRepository = SampleRepositories.BotTestRepository.IssuesResource();
+
+            NotExistingRepository = SampleRepositories.BotTestRepository.IssuesResource();
+        }
+
+        [Test]
+        public void ListIssues_ExistingPublicRepositoryWithIssue_ReturnValidInfo()
+        {
+            var issues = ExistingRepository.ListIssues();
+
+            issues.ShouldNotBeNull();
+            issues.Count.ShouldBe(2, "Only two issues are known on BotTestRepository");
+        }
+
+        [Test]
+        public void ListIssues_ExistingPublicRepositoryWithIssue_OnlyOpen_ReturnValidInfo()
+        {
+            var parameters = new ListIssuesParameters
+            {
+                Statuses = new[] { IssueStatus.Open },
+                Kinds = new[] { IssueKind.Task }
+            };
+
+            var pullRequests = ExistingRepository.ListIssues(parameters);
+            pullRequests.ShouldNotBeNull();
+            pullRequests.Count.ShouldBeGreaterThan(0, "When we don't limit to open pull request we can found more!");
```

Comment:
Assertion to rework:
Message is a copy/past that has been forgotten to be adapted
And I would except a known count of 1 to show that it has effectively filter the results

Indeed, this test should bring some light on the comment I did on ListIssuesParameters ;)

Reply:
Fixed.